src/fmu/ensemble/realization.py

```diff
@@ -1166,7 +1160,6 @@ def static_get_volumetric_rates(realization, column_keys, time_index, time_unit)
         This method is to be used by both ScratchRealization
         and VirtualRealization, and is documented there."""
         import calendar
```

Comment:
you don't need a `# pylint: disable=import-outside-toplevel` here?

Reply:
Moved to top of file, no circular import problems here.